eng/pipelines/templates/stages/vmr-build.yml

```
@@ -390,14 +390,14 @@ stages:
       #     targetOS: linux
       #     targetArchitecture: arm64
 
-      # - template: ../jobs/vmr-build.yml
-      #   parameters:
-      #     buildName: OSXCross
-      #     isBuiltFromVmr: ${{ parameters.isBuiltFromVmr }}
-      #     vmrBranch: ${{ variables.VmrBranch }}
-      #     architecture: arm64
-      #     pool:
-      #       vmImage: ${{ variables.defaultPoolNameMac }}
-      #     container: ''
-      #     targetOS: osx
-      #     targetArchitecture: arm64
+        - template: ../jobs/vmr-build.yml
+          parameters:
+            buildName: OSX_x64_Cross
```

we're building on arm64 Mac now internally so this is no longer accurate. I'd remove it:
```suggestion
            buildName: OSX
```